odd_collector/adapters/redshift/mappers/tables.py

```diff
@@ -14,16 +14,21 @@
 from .views import extract_transformer_data
 from typing import List
 
+from ...postgresql.mappers.models import PrimaryKey
```

Comment:
Let's not use dependencies from another modules like this.
We need to have common modules for such cases

@Vixtir FYI